core/src/conn.rs

```diff
@@ -133,6 +138,47 @@ impl MongoConnection {
         }
     }
 
+    fn get_libmongosqltranslate_version() -> Result<String> {
+        let get_mongosqltranslate_version_command = doc! {
+            "command": "getMongosqlTranslateVersion",
+            "options": {},
+        };
+
+        let returned_doc =
+            handle_libmongosqltranslate_command(get_mongosqltranslate_version_command)?;
+
+        let libmongosqltranslate_version = returned_doc
+            .get("version")
+            .expect(
+                "The `version` key is missing from the document returned by libmongosqltranslate.",
+            )
+            .as_str()
+            .expect("The `version` key should map to a `String`")
+            .to_string();
```

Comment:
I added an error here instead. I think it's better if we have an error instead of attaching an unknown version because the version should always be present.